.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Question
+about: Ask a question
+labels: 'question'
+
+---
+
+## Background [Optional]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)